src/app/components/view-request-page/view-request-page.component.html

```diff
@@ -11,18 +11,11 @@ <h1>Chat Request</h1>
             {{chatrequest.time}}
             {{chatrequest.date}}
           </p>
-          <div *ngIf="chatrequest.isPending == true" class="isPending">
             <button class="btn btn-sm btn-success" (click)="approveUserChatRequest(chatrequest)">
-              Give Access
```

Comment:
refactor this line, i think after getting approval also it will raise the ticket again